show/main.py

```diff
@@ -130,6 +133,32 @@ def run_command(command, display_cmd=False, return_cmd=False):
     if rc != 0:
         sys.exit(rc)
 
+def run_command_pipe(*args, display_cmd=False, return_cmd=False):
```

Comment:
>run_command_pipe [](http://example.com/codeflow?start=4&length=16)

Possible to unify `run_command` and `run_command_pipe` into one function `run_command`?

Or treat `run_command` as a special case, call the other with only one command in args, so most of the code are reused.